src/exodus/sigmadb.cpp

```diff
@@ -285,6 +294,71 @@ void CMPMintList::RecordMintKey(const leveldb::Slice& mintKey)
     }
 }
 
+void CMPMintList::RecordGroupSize(uint16_t groupSize)
+{
+    auto key = CreateGroupSizeKey();
+
+    exodus::swapByteOrder(groupSize);
```

Comment:
The other values was not stored as big-endian. So we have 2 choices:

1. Remove this line and stored as little-endian.
2. Store all other values as big-endian.

Reply:
agree with 1)